linkify-item-names.user.js

```diff
@@ -132,6 +137,16 @@ function linkify_items_in_syndicate_campaign_rewards() {
     }
 }
 
+function store_item_params(slug) {
+    var item_type = $('.data-list>.type>span');
```

Comment:
For performance: We should store this into a variable (say, `var data_list = $('.data-list')`), then use `data_list.find('...')` below, so that the jQuery calls below aren't querying the entire DOM from scratch each time.